tests/unit_tests/test_runner.py

```
@@ -235,92 +256,120 @@ def test_save_final_model(tmp_path, mgf_small, tiny_config):
-    result_file = tmp_path / "result.mztab"
     with ModelRunner(
         config, model_filename=str(model_file), overwrite_ckpt_check=False
     ) as runner:
         runner.predict([mgf_small], result_file, evaluate=True)
 
     assert result_file.is_file()
-    result_file.unlink()
 
+
+@pytest.mark.parametrize(
+    "input_files, expect_match",
```

Ruff (0.15.2)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)